packages/components/teritoriNameService/NameDataForm.tsx

```diff
@@ -120,100 +124,106 @@ export const NameDataForm: React.FC<{
       />
       <TextInputCustom<Metadata>
```

Comment:
Already said that in an other PR for the same component, i don't ask to change it right now because it's not the purpose of the PR, but can you let a comment to say that it'll need to be refacto in a PR that will improve the `TextInput` uses ? (In every file you use it, to don't forget)

Reply:
Done.
Yes, TextInputCustom is too heavy